### PR TITLE
THREESCALE-8071: Clarified the prerequisites about the usage of OIDC with WebAssembly and 3scale versions.

### PR DESCRIPTION
**Pull request by dfennessy**
Work done for Jira Issue [THREESCALE-8071](https://issues.redhat.com/browse/THREESCALE-8071)

Related GitHub Issue: https://github.com/openshift/openshift-docs/issues/40406

**Reviewers**

* @JStickler 
* @rahulanand16nov (3scale SME)
* @pehala (3scale QE)

### CODE REVIEW

File: service_mesh/v2x/ossm-threescale-webassembly-module.adoc [hunk 1]
```diff
@@ -29,11 +29,16 @@ The `threescale-wasm-auth` module is designed to be fully compatible with all im
...
+[NOTE]
+====
+The module works with all supported 3scale releases prior to 2.11 except when configuring a service to use OpenID Connect (OIDC).
+====
+
 [id="prerequisites_ossm-threescale-webassembly-module"]
 == Prerequisites
 
-* The module works with all supported 3scale releases except when configuring a service to use xref:../../authentication/identity_providers/configuring-oidc-identity-provider.adoc#configuring-oidc-identity-provider[OpenID Connect (OIDC)].
 * For this WebAssembly configuration, you will need 3scale 2.11 or later.
```
Comment (rahulanand16nov) [inline service_mesh/v2x/ossm-threescale-webassembly-module.adoc:40]: I am not sure what `this` configuration means here?

Comment (rahulanand16nov) [reply service_mesh/v2x/ossm-threescale-webassembly-module.adoc:40]: Shouldn't this be along the lines of
```
 The module works with all supported 3scale releases, except when configuring a service to use OpenID Connect (OIDC), which requires _3scale 2.11 or later_
```

**Comment by rahulanand16nov** [comment]
LGTM

cc @unleashed

**Review by rahulanand16nov — APPROVED** [review]

**Review by pehala — APPROVED** [review]
LGTM

**Comment by dfennessy** [comment]
@JStickler 

I believe this is ready to be merged. Cheers.

**Comment by dfennessy** [comment]
cherrypick to enterprise-4.6, enterprise-4.7, enterprise-4.8, enterprise-4.9, enterprise-4.10

**Review by JStickler — APPROVED** [review]
LGTM

**Comment by dfennessy** [comment]
Thank you @JStickler :tada:

**Pull request merged**